_tests/browser_tests/.eslintrc

```diff
@@ -0,0 +1,10 @@
+env:
+  node: true
+  jasmine: true
+rules:
+  no-unused-expressions: 0
+  no-process-env: 0
+globals:
+  by: true
+  browser: true
```

Comment:
Oh yes, see now that there is a `browser` variable. That's kind of confusing, but oh well.